board/common/rootfs/usr/libexec/infix/init.d/05-product

```diff
@@ -1,22 +1,44 @@
 #!/bin/sh
-# Find and install any product specific files in /etc before bootstrap
+# Find, install, and run product specific files and script in /etc
+# before resuming bootstrap.
+#
+# Use /etc/product/init.d/S01-myscript for scripts, may be a symlink, it
+# will be called with `start` as its only argument.
+#
+# The compatible array is listed in reverse order, least significant to
+# most significant.  We apply all in order so the most significant can
+# override more generic ones.
 ident=$(basename "$0")
 
+PRODUCT_INIT=/etc/product/init.d
 PREFIXD=/usr/share/product
-PRODUCT=$(jq -r '."product-name" | ascii_downcase' /run/system.json)
+COMPATIBLES=$(jq -r '."compatible"[] | ascii_downcase' /run/system.json)
```

Comment:
```suggestion
COMPATIBLES=$(jq -r '."compatible" | reverse.[] | ascii_downcase' /run/system.json)
```
...i.e., reverse it here instead

Reply:
Sure, I can swap it around.  Then we can use `compatible[o]` as the `product_id`.